torch_xla/core/xla_env_vars.py

```diff
@@ -31,3 +31,4 @@
 WORLD_SIZE = 'WORLD_SIZE'
 LOCAL_WORLD_SIZE = 'LOCAL_WORLD_SIZE'
 ZERO_COPY_ENABLED = 'ZERO_COPY_ENABLED'
+XLA_FALLBACK_CUDA = 'XLA_FALLBACK_CUDA'
```

Comment:
I think it makes sense to defaul XLA:CUDA executions on CUDA fallback, while XLA:CPU and XLA:TPU remain on CPU fallback.

Reply:
As discussed offline, more specifically, after this PR, it would be good to:

Step1: Reversing the function of `XLA_FALLBACK_CUDA` so the default would be CUDA fallback enabled (e.g. define `DISABLE_XLA_FALLBACK_CUDA`). 

Step2: define a mechanism to remove the env variable and have a native solution that just works, and avoids user experience hiccups.